src/SolutionTemplate/UnoSolutionTemplate.Wizard/UnoSolutionWizard.cs

```diff
@@ -35,16 +36,54 @@ public void RunFinished()
 
 			if (!File.Exists(nugetConfigPath))
 			{
-				using (var reader = new StreamReader(GetType().Assembly.GetManifestResourceStream($"{GetType().Assembly.GetName().Name}..vsconfig")))
+				using var reader = new StreamReader(GetType().Assembly.GetManifestResourceStream($"{GetType().Assembly.GetName().Name}..vsconfig"));
+				File.WriteAllText(nugetConfigPath, reader.ReadToEnd());
+			}
+
+			OpenWelcomePage();
+			SetAsStartupProject();
+		}
+
+		private void OpenWelcomePage()
+			=> _dte?.ItemOperations.Navigate("https://platform.uno/docs/articles/getting-started-tutorial-1.html", vsNavigateOptions.vsNavigateOptionsNewWindow);
+
+		private void SetAsStartupProject()
+		{
+			if (_dte != null && _dte.Solution.SolutionBuild is SolutionBuild2 val)
+			{
+				try
+				{
+					var uwpProject = _dte.Solution.Projects.Cast<Project>().FirstOrDefault(s => s.Name.EndsWith(".UWP", StringComparison.OrdinalIgnoreCase));
+
+					if (uwpProject is { })
+					{
+						val.StartupProjects = uwpProject.UniqueName;
+					}
+
+					var x86Config = val.SolutionConfigurations
+						.Cast<SolutionConfiguration2>()
+						.FirstOrDefault(c => c.Name == "Debug" && c.PlatformName == "x86");
+
+					x86Config?.Activate();
+				}
+				catch (Exception)
```

Comment:
Codacy found an issue: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6286216)